modules/client-configuration/pages/clients-almalinux.adoc

```diff
@@ -3,12 +3,26 @@
 
 This section contains information about registering Salt clients running {almalinux} operating systems.
 
-[NOTE]
+ifeval::[{suma-content} == true]
+[IMPORTANT]
+====
+{amazon} repository URLs are available from {scc}, but the packages and metadata are provided by Amazon, not by {suse}.
```

Comment:
Huh? Amazon? Also this is unrelated to this PR, but OK.

Reply:
I still see Amazon here. Amazon does not provide packages or metadata for AlmaLinux :-)